modules/govuk/manifests/apps/publishing_api.pp

```diff
@@ -143,6 +152,12 @@
       port => $redis_port,
     }
 
+    govuk::app::envvar::rabbitmq { $app_name:
+      hosts    => $rabbitmq_hosts,
+      user     => $rabbitmq_user,
+      password => $rabbitmq_password,
```

Comment:
Where was this set previously, as the parameter already existed?

Reply:
It was specified below, which I've now removed. Nice catch.